app/views/accounts/new/_method_selector.html.erb

```diff
@@ -6,16 +6,27 @@
       <span class="flex w-8 h-8 shrink-0 grow-0 items-center justify-center rounded-lg bg-alpha-black-50 shadow-[inset_0_0_0_1px_rgba(0,0,0,0.02)]">
         <%= lucide_icon("keyboard", class: "text-gray-500 w-5 h-5") %>
       </span>
-      <%= t("accounts.new.method_selector.manual_entry") %>
+      <%= t(".manual_entry") %>
```

Comment:
Since this partial is shared across all accountables (delegated types), we need to keep the path absolute:

`accounts.new.method_selector.manual_entry`

That should fix the e2e tests